### ISSUE TITLE
[Keyboard Navigation-Home Page-Blog]: Focus is moving to some hidden link showing on left panel on Blog page

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2649]
**User Experience:** 
User who depends on Keyboard with screen reader may get misguided about the links and controls if focus moves to any hidden link on Blog Page.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps:-**
1. Navigate to https://makecode.com/ 
2. Navigate to the Blog in header section and select it.
3. Navigate to the Blog page using tab key.
6. Verify the focus order on the Blog page.
 
**Actual Result:-**
While navigating the page using tab key Focus is moving to some hidden link showing on left panel on Blog page.

**Expected Result:-**
While navigating the page using tab key Focus should not move to any hidden link showing on left panel on Blog page

**Note:** This issue exists with Firefox also.
 
**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Suggested Fix:**
tab index property is applicable to all visual elements on screen.  
<a href="/articles/" tab index="2">Articles</a> 
<a href="/resources/" tab index="3">Resources</a> 
 
Tips : 
'tab index' should only be used in cases where: 
•The default tab order is not ideal, AND 
•The tab order cannot be changed by rearranging items in the content and/or by altering the style sheet to reflect the best visual arrangement. 
 
Key References for Fix: 
http://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20160317/G59 
https://www.w3.org/WAI/GL/wiki/Creating_Logical_Tab_Order_with_the_Tabindex_

**Please refer the Attachment:**
[MAS2.4.3_Home page_Blog.zip](https://github.com/Microsoft/pxt/files/1196268/MAS2.4.3_Home.page_Blog.zip)

**Comment by v-mesala** [comment]
Bug is verified on URL : https://makecode.com/ and bug still exists .Hence Reactivated the bug and added the Active tag.

**Comment by v-mesala** [comment]
Bug is verified on URL : https://makecode.com/ and bug still exists .Hence Reactivated the bug removed the tag Fixed and added the Active tag.

**Comment by veler** [comment]
The issue is fixed locally. It is not deployed yet on 8/28. It will be available to test soon.

**Comment by veler** [comment]
Fix available on [https://makecode.microbit.org/](https://makecode.microbit.org/)

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.com/ and issue is fixed now. Hence closing the bug
